windows/setup.yml

```diff
@@ -306,3 +352,139 @@ controller_templates:
           type: text
           variable: HOSTS
           required: false
+
+controller_workflows:
+  - name: Setup Active Directory Domain
+    description: A workflow to create a domain controller with two domain-joined Windows hosts.
+    organization: Default
+    notification_templates_started: Telemetry
+    notification_templates_success: Telemetry
+    notification_templates_error: Telemetry
+    survey_enabled: true
+    survey:
+      name: ''
+      description: ''
+      spec:
+        - question_name: AWS Region
+          type: multiplechoice
+          variable: create_vm_aws_region
+          required: true
+          default: us-east-2
```

Comment:
in other surveys i don't think we set a default for AWS region, but i like the idea so i'll put in an issue to do so.